### PR TITLE
Pull Request to Golden Repo

### PR DESCRIPTION
**Pull request by zdehkordi**
Thanks for looking over our work before final submit

### CODE REVIEW

File: app/controllers/charges_controller.rb [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+       else
+        return [day_index, month_index, year_index]
+       end
+    end
+
+    public 
+    def new
+    end
+    
+    def create
```
Comment (adnanhemani) [inline app/controllers/charges_controller.rb:105]: This method is a beast. Please refactor - consider breaking up into more methods if needed

File: app/controllers/calendars_controller.rb [hunk 2]
```diff
@@ -0,0 +1,213 @@
+class CalendarsController < ApplicationController
```
Comment (adnanhemani) [inline app/controllers/calendars_controller.rb:1]: Was goign through your Heroku deployment - couldn't see anywhere where a coach could input their availability?

File: app/controllers/charges_controller.rb [hunk 3]
```diff
@@ -0,0 +1,243 @@
+class ChargesController < ApplicationController
```
Comment (adnanhemani) [inline app/controllers/charges_controller.rb:1]: Getting this error when running Rspec. How do I resolve?
Screen Shot 2019-05-12 at 2 18 33 PM

**Review by adnanhemani — CHANGES_REQUESTED** [review]
Overall, well done! Just need to resolve the comments and y'all should be good to go.

File: app/controllers/charges_controller.rb (hunk 3 above)
Comment (chenjoe569) [reply app/controllers/charges_controller.rb]: You need to decrypt it by doing this: 
gpg --passphrase "getrekt" --decrypt --output config/application.yml config/application.yml.asc

File: app/controllers/calendars_controller.rb (hunk 2 above)
Comment (chenjoe569) [reply app/controllers/calendars_controller.rb:1]: It should be in the profile page

Comment (chenjoe569) [reply app/controllers/calendars_controller.rb:1]: Go to My account drop down, and then click view profile, then you will see Add availability option

File: app/controllers/charges_controller.rb (hunk 1 above)
Comment (chenjoe569) [reply app/controllers/charges_controller.rb:105]: Ok we will try, though this we honestly already kinda cracked our heads trying to make it simpler, but still couldn’t. We will try to refactor a bit more

Comment (adnanhemani) [reply app/controllers/charges_controller.rb:105]: I'm sure that there's some logical breaks where you can extract and call it something different. Also, most of the work should be done in a model, not controller. So if anything else, break it up and move the pieces to model

File: app/controllers/calendars_controller.rb (hunk 2 above)
Comment (adnanhemani) [reply app/controllers/calendars_controller.rb:1]: OH I see. Good job!

File: app/controllers/charges_controller.rb (hunk 3 above)
Comment (adnanhemani) [reply app/controllers/charges_controller.rb:1]: You shouldn't have put the password here....this is a public repo

Invalidate all API keys in this file, generate new ones, re-encrypt the file, and then securely send me the keys. Email me, I'll send you my GPG key.

Comment (chenjoe569) [reply app/controllers/charges_controller.rb:1]: Sorry, my bad. I deleted the comments. Though the api key encrypted that I provided is just my test key that I personally created. I’ll do this now.

**Review by adnanhemani — APPROVED** [review]

**Pull request merged**